code/game/machinery/machinery.dm

```diff
@@ -245,10 +254,6 @@
 	if(!interact_offline && stat & (NOPOWER|BROKEN|MAINT))
 		return TRUE
 
```

Comment:
please document these procs really well, coders should really understand why and when to use `try_attack_hand`